src/client/app/admin/authentication/authentication.service.ts

```diff
@@ -50,7 +51,7 @@ export class AuthenticationService {
 	};
 
 	// Get the user who is currently logged in (or null if no one is logged in)
-	public getCurrentUser(): User {
+	public getCurrentUser() {
```

Comment:
Issues with having User be required. It's probably fine now to add it back in but anything to do with AuthenticationService has been pretty painful to load elsewhere.